pkg/daemons/control/server.go

```diff
@@ -288,6 +301,10 @@ func prepare(ctx context.Context, config *config.Control, runtime *config.Contro
 	runtime.ClientAuthProxyCert = path.Join(config.DataDir, "tls", "client-auth-proxy.crt")
 	runtime.ClientAuthProxyKey = path.Join(config.DataDir, "tls", "client-auth-proxy.key")
 
+	if err := os.Setenv("KUBE_CONFIG_ADMIN", runtime.KubeConfigAdmin); err != nil {
+		return err
```

Comment:
Is this still needed? It seems strange passing runtime.KubeConfigAdmin into the cloud controller when runtime.KubeConfigCloudController is also created.